python/cudf/bindings/join.pyx

```diff
@@ -13,16 +13,17 @@ from .cudf_cpp import *
 import numpy as np
 import pandas as pd
 import pyarrow as pa
-
-cimport numpy as np
+import numpy as np
 
 from librmm_cffi import librmm as rmm
 
 from libc.stdint cimport uintptr_t
 from libc.stdlib cimport calloc, malloc, free
 
+cimport cython
 
 
+@cython.boundscheck(False)
```

Comment:
Why are we disabling bounds-checking?

Reply:
To disable a warning that appeared when this PR changed the way we created NumPy arrays.

Reply:
That makes sense. Thanks for clarifying.